docs/Quality.md

```diff
@@ -43,11 +43,36 @@ Utilities like [Istanbul](https://github.com/gotwarlost/istanbul) provide the pe
 	* [BEHAT](https://www.drupal.org/project/drupalextension)
 	* [phpunit](https://phpunit.de)
 
-## QA?
-TODO
+## QA
+QA can take different forms. At the most basic level, a QA team can perform manual testing. This involves writing manual scripts, executing them, and then filing bugs when issues are encountered while executing those scripts. Manual testing is time consuming, and is still prone to human error.
+
+A QA team can also write automated tests, even down to the unit level. However, developers tend to be best suited for unit and most integration testing, since those benefit from a deeper understanding of what will be written. A QA Engineer is well suited for writing and maintaining end-to-end tests though, since those focus on whether acceptance criteria has been satisified or not.
```

Comment:
"...acceptance criteria has been..." should read: "...acceptance criteria have been..." assuming criteria is the plural form of 'criterion'